CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Welcome to the i18n Translation Generator contributing guide
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```

Comment:
```suggestion
Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```

Not sure if this is accurate? Maybe this statement is valid in another repo

Reply:
Thanks! My mistake